src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -175,20 +174,6 @@ def is_closed(self) -> bool:
     def get_session_id(self) -> int:
         return self._conn.session_id
 
-    def get_default_database(self) -> Optional[str]:
-        return (
-            quote_name(self._lower_case_parameters["database"])
-            if "database" in self._lower_case_parameters
-            else None
-        )
-
-    def get_default_schema(self) -> Optional[str]:
-        return (
-            quote_name(self._lower_case_parameters["schema"])
-            if "schema" in self._lower_case_parameters
-            else None
-        )
-
```

Comment:
These two methods are not used at all. Since they are pure internal functions, instead of testing them, I think we should just get rid of them.